src/main/kotlin/com/sourcegraph/cody/config/SettingsMigration.kt

```diff
@@ -118,6 +129,54 @@ class SettingsMigration : Activity {
         }
   }
 
+  fun migrateDeprecatedChatLlms(project: Project) {
```

Comment:
btw, is it possible to write a test for it easily?

Reply:
I added a test